optimize/self-managed/optimize-deployment/migration-update/3.10-to-3.11_8.3.md

```diff
@@ -1,13 +1,13 @@
 ---
-id: 3.10-to-7.20_8.3
-title: "Update notes (3.10 to 7.20/8.3)"
+id: 3.10-to-3.11_8.3
```

Comment:
I am pretty sure I added a redirect to this file in `.htaccess`. Does it need changing/removing?

Reply:
Oops! We shouldn't need it, but I'll have a look.